content/doc/firmware/debugging.md

```diff
@@ -129,6 +129,26 @@ Example of output:
 # 8.50 <I> In application task
 ```
 
+## Read log
+
+Read log with connected device and following command:
+```
+bcf log --device [device]
+```
+
+[device] port can be founded by command:
+```
+bcf log devices
+```
+Example of outuput:
+```
+# 0.50 <I> In application task
```

Comment:
Please put there examples of DUMP, DEBUG, INFO, WARNING, ERROR altogether with some nice screenshot from colored terminal :smile: Thanks!